### PR TITLE
Bringing in 52North WPS project

### PR DESCRIPTION
**Pull request by isuftin**

### CODE REVIEW

**Comment by isuftin** [comment]
Currently, the 52N repository is down so this will not build if merged. This should be merged once the repository comes back up

File: dsas-n52/src/main/webapp/config/wpsCapabilitiesSkeleton.xml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        <ows:ServiceTypeVersion>1.0.0</ows:ServiceTypeVersion>
+        <ows:ServiceTypeVersion>0.4.0</ows:ServiceTypeVersion>
+        <ows:Fees>NONE</ows:Fees>
+        <ows:AccessConstraints>NONE</ows:AccessConstraints>
+    </ows:ServiceIdentification>
+    <ows:ServiceProvider>
+        <ows:ProviderName>CIDA</ows:ProviderName>
+        <ows:ProviderSite xlink:href="http://cida.usgs.gov/"/>
+        <ows:ServiceContact>
+            <ows:IndividualName>Jordan Read</ows:IndividualName>
```
Comment (jiwalker-usgs) [inline dsas-n52/src/main/webapp/config/wpsCapabilitiesSkeleton.xml]: put Megan here?

Comment (isuftin) [inline dsas-n52/src/main/webapp/config/wpsCapabilitiesSkeleton.xml]: Once we get everything moved :D

File: dsas-n52/src/main/webapp/config/wps_config.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            <Property name="Rserve_Password" active="true">need_a_password</Property>
+            <Property active="true" name="Script_Dir">R/r_scripts</Property>
+            <Property active="true" name="Resource_Dir">R/resources</Property>
+            <Property name="R_Work_Dir" active="true">default</Property>
+            <Property name="Enable_Batch_Start" active="true">false</Property>
+        </Repository>
+    </AlgorithmRepositoryList>
+    <!--@hostname@, @hostport@ and @webappPath@ are filtered and replaced by values derived from
+    HttpServelRequest instance of getRequestURL() in response using a Filter-->
+    <Server hostname="cida-wiwsc-cchdev.er.usgs.gov" hostport="8080" webappPath="wps" includeDataInputsInResponse="true" computationTimeoutMilliSeconds="5" cacheCapabilites="false">
```
Comment (jiwalker-usgs) [inline dsas-n52/src/main/webapp/config/wps_config.xml]: If we update to a new version of n52, there are more attributes needed for this element

**Pull request merged**